modules/pico_tcp.c

```diff
@@ -54,12 +54,16 @@
 
 
 #ifdef PICO_SUPPORT_TCP
-#define tcp_dbg_nagle(...) do {} while(0)
-#define tcp_dbg_options(...) do {} while(0)
-
 
-#define tcp_dbg(...) do {} while(0)
-/* #define tcp_dbg dbg */
+#ifdef DEBUG_TCP
+#define tcp_dbg_nagle        dbg
+#define tcp_dbg_options      dbg
+#define tcp_dbg              dbg
```

Comment:
MM : I don't think so, that looks more like "I want colors in my debugging" than "I want different levels of debugging info" - could be wrong though.
tap device : yes, looks like it would be good to do it there as well.